ScoutSuite/providers/aws/configs/regions_config.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from ScoutSuite.providers.base.configs.resources import Resources
+from ScoutSuite.providers.aws.facade import AWSFacade
+
+
+class RegionsConfig(Resources):
```

Comment:
Don't you want to rename this class `Regions`? (Just for being consistent with `Resources`, `Lambdas` and `RegionalLambdas`.)

Reply:
It was renamed in my EC2 PR <3